src/main/java/org/jboss/arquillian/testcontainers/TestContainersObserver.java

```diff
@@ -42,7 +42,14 @@ public void createContainer(@Observes(precedence = 500) BeforeClass beforeClass)
         final DockerRequired dockerRequired = javaClass.getAnnotation(DockerRequired.class);
         if (dockerRequired != null) {
             if (!isDockerAvailable()) {
-                throw createException(dockerRequired.value());
+                // TODO Discuss non-WildFly options
+                if ("true".equals(System.getProperty("org.wildfly.test.require.docker")) ||
```

Comment:
Right... this is the problematic bit that I mentioned we need to find a generic solution to.

Reply:
+1 having a WildFly property here doesn't feel right to me.

Reply:
I added that solely as a backwards-compatible option so we wouldn't have to change all of our jobs, but, yes. It feels gross. I'll remove it.